packages/next/build/babel/plugins/next-page-config.ts

```diff
@@ -102,6 +113,11 @@ export default function nextPageConfig({
                       config[name] = prop.value.value
```

Comment:
Suggestion that would allow shorthands, methods, variable references, etc:

```js
Object.assign(config, eval('(' + declaration.init.getSource() + ')'));
```

Reply:
We intentionally want this to be very "dumb" and force users to use the desired data type. We'll note this for the future, though!